src/main/java/Main.java

```diff
@@ -28,16 +30,28 @@ public class Main {
     private static final String GENERATOR_STANDARD_FILES_PATH = "src/main/resources/standard_files";
 
     public static void main(String[] args) throws IOException {
-        //File file = new File ("./src/main");
-        //CreateProjectStructureAsJson createProjectStructureAsJson = new CreateProjectStructureAsJson(file);
+        //  File file = new File ("./src/main/java/util");
+        // UserCodeResolver createProjectStructureAsJson = new UserCodeResolver(file);
+    
+        String groupIdPart1 = "de";
```

Comment:
Haben wir alles in Variablen außerhalb der main-Methode. Kann also gelöscht werden